org.deidentifier.arx.yaml

```
@@ -0,0 +1,62 @@
+app-id: org.deidentifier.arx
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+
+command: arx
+finish-args:
+  - --socket=x11
+#  - --device=dri
+  - --share=ipc
+  # network: the app can connect to DBMS
+  - --share=network
+  - --env=PATH=/usr/bin:/app/bin:/app/jre/bin
+  - --env=JAVA_HOME=/app/jre
+
+build-options:
+  env:
+    JAVA_HOME: /usr/lib/sdk/openjdk17/
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+
+  - name: arx-binary
+    buildsystem: simple
+    build-commands:
+        - install -D -ma+rx arx.sh /app/bin/arx
+        - install -D -ma+r arx-gtk-64.jar /app/bin/arx-gtk-64.jar
+        - install -D -ma+r org.deidentifier.arx.desktop /app/share/applications/org.deidentifier.arx.desktop
+        - install -D -ma+r org.deidentifier.arx.appdata.xml /app/share/metainfo/org.deidentifier.arx.appdata.xml
+        - install -D -ma+r org.deidentifier.arx.svg /app/share/icons/hicolor/scalable/apps/org.deidentifier.arx.svg
+        - install -D -ma+r org.deidentifier.arx.png /app/share/icons/hicolor/128x128/apps/org.deidentifier.arx.png
```

```suggestion
        - install -D -ma+r org.deidentifier.arx.svg /app/share/icons/hicolor/scalable/apps/org.deidentifier.arx.svg
```
You don't need the PNG if you have a SVG

---

the SVG look off-center.

---

yeah, the quality of the file is not good. That's why I prefer the rasterised version. Maybe @prasser can look at the SVG file and make it look nice.